hrjobcontract/js/test/controllers/contract-list_test.js

```diff
@@ -0,0 +1,106 @@
+define([
+  'job-contract/app'
+], function() {
+  'use strict';
+
+  describe('ContractListCtrl', function() {
+    var ctrl, $rootScope, $controller, $scope, $q, $httpBackend, ContractHealthService;
+
+    beforeEach(module('hrjc'));
+
+    beforeEach(module(function ($provide) {
+      $provide.factory('ContractHealthService', function() {
+        return {
+          getOptions: function () {},
+          getFields: function () {}
+        }
+      });
+    }));
+
+    beforeEach(inject(function (_$controller_, _$rootScope_, _$q_, _$httpBackend_, _ContractHealthService_) {
+      $controller = _$controller_;
+      $rootScope = _$rootScope_;
+      $httpBackend = _$httpBackend_;
+      $q = _$q_;
+      ContractHealthService = _ContractHealthService_;
+
+      contractHealthServiceSpy()
+
+      makeController();
+    }));
+
+    beforeEach(function () {
+      $httpBackend.whenGET(/action=get&entity=HRHoursLocation/).respond({});
+      $httpBackend.whenGET(/action=get&entity=HRPayScale/).respond({});
+      $httpBackend.whenGET(/action=getfields&entity=HRJobDetails/).respond({});
+      $httpBackend.whenGET(/action=getfields&entity=HRJobHour/).respond({});
+      $httpBackend.whenGET(/action=getfields&entity=HRJobPay/).respond({});
+      $httpBackend.whenGET(/action=getfields&entity=HRJobLeave/).respond({});
+      $httpBackend.whenGET(/action=getfields&entity=HRJobHealth/).respond({});
+      $httpBackend.whenGET(/action=getfields&entity=HRJobPension/).respond({});
+      $httpBackend.whenGET(/action=get&entity=HRJobContract/).respond({});
+      $httpBackend.whenGET(/action=getoptions&entity=HRJobHealth/).respond({});
+      $httpBackend.whenGET(/views.*/).respond({});
+    });
+
+    describe("Fetch Health Plans Options on opening new ontract modal", function () {
```

Comment:
This is the test structure you should strive for
```js
describe('contract modal', function () {

  describe('on init', function () {

    it('fetches the health plan options', function () {

    });
  });
});
```
notice that it reads as an english sentence, making the test very easy to follow. and also it makes each section (each `describe`, that is) easy to extend in case you need to add more tests and/or sections